### DISCUSSION TITLE
Recommendation: Do not run any Sling supplied software

### DISCUSSION THREAD

**Discussion by GerryDazoo** [GerryDazoo/Slinger#281] [General]
In particular, any version of Desktop and especially do not try to make any configuration changes unless you know for sure you are completely disconnected from the Internet.

That software can reflash your box if you're not careful.

So far, only Solo, Pro and ProHD models have been affected but until things become a little clearer about what's happening better safe than sorry.

**Comment by tom76017** [comment]
Good point.  I found this string in the Desktop exe.   
...http//autoupdate.sling.com/plugin_binary/downloads/SPD/Win/Appcast/...

**Reply by mdehrlich** [reply]
Warning! I set my firewall to block my slingboxes from any access over the internet.  It is logging about 10 outbound attempts per minute for each slingbox.   It is using many different IP addresses.    If every box still active is banging on these addresses forever, I would expect that they are going to try to shut down all of us!  SET UP YOUR ROUTER FIREWALL!

**Comment by Mikoshi-nyudo** [comment]
Given this is a Slinger discussion, I'm not suggesting to use any SP application for viewing. Personally, for the bulk of users the Slinger is the safest way to go i.e. lockdown SB using FW rules, and have Slinger on LAN, then expose it to the WAN with Port forwards for remote viewing while the SB is hidden. I believe it's better for the average person, and you'll get support on how to setup and ensure things are done correctly.

However, I'll share this for the other aspects of SBCore.ini in 1.5 setup assistant, and other safety measures using hosts etc.. Hope that's ok Gerry?

https://www.reddit.com/r/slingbox/comments/yt6wpb/comment/iw4v7x2/?utm_source=share&utm_medium=web2x&context=3
